<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Legisladores</class>
 <widget class="QMainWindow" name="Legisladores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Legisladores</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="lwLegisGood">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>141</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lwLegisBad">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>141</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Legisladores Buenos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Legisladores Malos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Id:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tId">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>30</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bInsertar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Insertar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bSuprimir">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Suprimir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bIsMember">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Es Miembro</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tConj">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ingrese &lt;span style=&quot; font-weight:600;&quot;&gt;0&lt;/span&gt; para la conjunto de legisladores, &lt;span style=&quot; font-weight:600;&quot;&gt;1&lt;/span&gt; para legisadores buenos y &lt;span style=&quot; font-weight:600;&quot;&gt;2&lt;/span&gt; para legisladores malos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Conjunto:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bRevisar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Revisar</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Respuesta:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tRespuesta">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="gbLey">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Leyes</string>
    </property>
    <widget class="QCheckBox" name="cbBuena">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Buena</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tLey">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ingrese &lt;span style=&quot; font-weight:600;&quot;&gt;0&lt;/span&gt; para la conjunto de legisladores, &lt;span style=&quot; font-weight:600;&quot;&gt;1&lt;/span&gt; para legisadores buenos y &lt;span style=&quot; font-weight:600;&quot;&gt;2&lt;/span&gt; para legisladores malos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Ley:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bCrear">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Crear</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbLegislar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>371</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Legislar</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>57</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Ley:</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lwLey">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>121</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="lwLegis">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>141</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Legisladores</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bAprobar">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>290</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Aprobar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>967</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
